src/planwise/component/datasets2.clj

```diff
@@ -57,6 +58,11 @@
   (db-find-sites (get-db store) {:dataset-id dataset-id
                                  :version version}))
 
+(defn filter-sites-by-tags
+  [sites tags]
+  (reduce
+   (fn [sites tag] (let [filtered-sites  (filter #(-> % :tags (includes? tag)) sites)] filtered-sites)) sites tags))
```

Comment:
This will not work correctly for tags which are a substring of another tag. Ie. `includes?` will return `true` for "foo" in the string "foobar baz".